src/main/resources/product-inventory-spec.yaml

```
@@ -0,0 +1,85 @@
+openapi: 3.0.4
+info:
+  title: Product Inventory API
+  version: 1.0.0
+  description: A simple API to manage products in a warehouse
+
+servers:
+  - url: http://localhost:8080/products
```

no need to include any mapping. only domain name is required
```suggestion
  - url: http://localhost:8080/
```